content/client-server-api/_index.md

```diff
@@ -764,7 +766,7 @@ auth by offering a stage with only the `m.login.dummy` auth type, but they
 must still give a 401 response to requests with no auth data.
 {{% /boxes/note %}}
 
-#### Example
+**Example**
```

Comment:
I downgraded this to a non-heading, because I feel like it doesn't need to appear in the ToC.